doc/whats-new.md

```diff
@@ -1,330 +1,385 @@
+
+There was a huge amount of work on the conditioned GMF calculator, for
+which we included a [comprehensive verification test suite]
+(https://github.com/gem/oq-engine/pull/8542)
+based on the tests for the [ShakeMap code by USGS]
+(https://usgs.github.io/shakemap/manual4_0/tg_verification.html)
+We also changed the calculator to compute the GMFs only on the hazard sites
+and not on the seismic stations.
+
+At user request, we changed the behavior of `scenario_risk` calculations
+to just print a warning in case of small hazard causing to losses, and not
+to raise an error. This is consistent with the behavior of `scenario_damage`
+calculations
+
+Finally, we added an environment variable OQ_SAMPLE_ASSETS which is useful when
+debugging calculations; for instance you can set OQ_SAMPLE_ASSETS=.001
+to reduce a calculation by 1000 times, which will be much faster and easier
+to debug.
+
+# Bug fixes and new checks
+
+We reimplemented the `minimum_magnitude` filter in terms of the
+mag-dependent-distance filter, which can also be used to implement a
+`maximum_magnitude` filter. The change eliminated a bunch of bugs and
+corner cases that kept creeping in.
+
+The engine was rounding longitude and latitude to 5 digits everywhere, except 
+when generating a grid from a region; this has been fixed, and we have full
+consistency now.
+
+The `truncation_level` parameter is now mandatory in classical and
+event based calculations. Before, if the user forgot to set it, a
+default value of 99 was used, most likely not what the users wanted.
+
+If the same parameter is present in different sections of the job.ini
+file, now the engine raises an error.
+
+If the user by mistake disable the statistics and does not specify
+`individual_rlzs=true`, now gets an early error; before, the engine
+was computing the individual realizations (which was an error).
+
+If the exposure contains empty asset IDs, now the engine raises a clear
+error.
+
+If the site model file does not contain a required site parameter,
+and the parameter is not set in the job.ini either, now a clear
+error is used instead of using incorrectly a NaN value.
+
+The New Zealand model uses the `CScalingMSR`
+magnitude-scaling-relationship. That caused a surprising error when
+using the `ps_grid_spacing` approximation due to the way the names of
+the CollapsedPointSources were generated. This is now fixed since the
+name does not contain anymore the name of the MSR.
+
+It is now possible to use a percent character in the job.ini file
+without interfering with the ConfigParser interpolation feature.
+
+The GMF importer was failing to import GMFs in CSV format with a
+`custom_site_id` field: this has been fixed.
+
+In case of errors, the engine was printing the same traceback two or
+even three times: this has been fixed.
+
+# oq commands
+
+We finally removed the command `oq celery`, since we have deprecated
+celery for over 5 years. We also removed all references to celery and
```

Comment:
Please check this. I still find several references to celery in the docs.

Reply:
Good catch. Fixed in https://github.com/gem/oq-engine/pull/8887